apps/summon-app/src/layouts/SummonerForm.tsx

```diff
@@ -36,9 +40,12 @@ const Main = styled.main`
 `;
 
 export const SummonerForm = () => {
+  const { provider, chainId } = useHausConnect();
   const methods = useForm({ mode: 'onTouched' });
   const handleFormSubmit = async (formValues: Record<string, unknown>) => {
```

Comment:
Maybe the types can be something like this? https://github.com/raid-guild/headline/blob/6b6ac5b00dbacff0253418c43abf7af0858f49d0/src/pages/PublishPage.tsx#L337

SubmitHandler<FieldValues> as the return type might resolve some of the issues lower down in the function call tree